src/main/java/nl/hsac/fitnesse/fixture/slim/DateFormatFixture.java

```diff
@@ -0,0 +1,73 @@
+package nl.hsac.fitnesse.fixture.slim;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+
+/**
+ * Fixture that helps converting dates to and from various formats. Intended to be used as a utility fixture i.e. as library
+ */
+
+public class DateFormatFixture extends SlimFixture {
+    private String dateFormat = "dd-MM-yyyy";
```

Comment:
This is true. My first impulse is to explicitly set the desired values from the wiki in such a scenario. But in saying that the current solution works just as well, it also allows an explicit set for people that don't like the defaults coded in the class.

BUT, on the other hand, it is hard to back to using the system locale (as far as I know there is no string you can pass to explicitly use the system default, you have to use the constants from code).

So my preference is still to have the system defaults in code, and use these as defaults. This will be more intuitive to 'first-time' users. Where 'power-users' like yourself can set explicit values to ensure the test runs the same on any environment.

You could consider adding an extra constructor which explicitly configures the format and timezone, so you can still [use the fixture and configure it in a single table row](http://fitnesse.org/FitNesse.FullReferenceGuide.UserGuide.WritingAcceptanceTests.SliM.ConstructorArguments) (instead of needing 3).

What do you think?